[bug] Error near the end of 'conan create' build on Windows using CMake/Ninja with Conan 2.0

Last issue blocking me from moving to Conan 2.0 (well, aside from working around the [CMake wrapper](https://github.com/conan-io/cmake-conan) not being supported yet!)

### Environment Details (include every applicable attribute)
  * Operating System+version: Windows 10.0.20348 Build 768
  * Compiler+version: MSVC 19.32.31329.0
  * Conan version: 2.0.0-beta1
  * Python version: 3.9.13
Environment is the `Windows-2022` VM in GitHub Actions:
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md

### Steps to reproduce (Include if Applicable)
1. `git clone https://github.com/ssrobins/conan-recipes.git`
1. `cd conan-recipes`
1. `git checkout conan_v2_bug_report`
1. `py scripts/conan_create_all.py windows`

The GitHub Actions CI run produces this error:
`ERROR: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\runneradmin\\.conan2\\p\\tmp\\eb5cc8e93d895b84\\b\\build'`
(full log in next section)

Recipe being used:
https://github.com/ssrobins/conan-recipes/blob/conan_v2_bug_report/recipes/box2d/conanfile.py

- Not reproducible on my Windows desktop, only the GitHub Actions CI environment
- Everything works great with conan 1.49.0 (see `main` branch of this repo)
- Not reproducible on Linux or macOS, just Windows
- Problem goes away when I stop forcing `Ninja Multi-Config` CMake generator, it then defaults to Visual Studio / msbuild

### Logs (Executed commands with output) (Include/Attach if Applicable)
```
conan create --update --user ssrobins . -pr:h=D:\a\conan-recipes\conan-recipes\scripts/../profiles/windows_x86 -pr:b=D:\a\conan-recipes\conan-recipes\scripts/../profiles/windows_x64 -pr:b=D:\a\conan-recipes\conan-recipes\scripts/../profiles/default -s build_type=Debug
-------- Exporting the recipe ----------
box2d/2.3.1@ssrobins: Exporting package recipe
box2d/2.3.1@ssrobins: Copied 1 '.txt' file: CMakeLists.txt
box2d/2.3.1@ssrobins exports_sources: Copied 1 '.txt' file: CMakeLists.txt
box2d/2.3.1@ssrobins: Using git commit as the recipe revision: 461952414da5d1f5af578bd53f97e2e0cc5babcf
box2d/2.3.1@ssrobins: A new conanfile.py version was exported
box2d/2.3.1@ssrobins: Folder: C:\Users\runneradmin\.conan2\p\ac46809199461bab\e
box2d/2.3.1@ssrobins: Exported revision: 461952414da5d1f5af578bd53f97e2e0cc5babcf
-------- Input profiles ----------
Profile host:
[settings]
arch=x86
build_type=Debug
compiler=msvc
compiler.cppstd=17
compiler.runtime=static
compiler.runtime_type=Debug
compiler.version=193
os=Windows
[options]
[tool_requires]
[env]
Profile build:
[settings]- Everything works great with conan 1.49.0 (see `main` branch of this repo)
arch=x86_64
[options]
[tool_requires]
[env]
-------- Computing dependency graph ----------
box2d/2.3.1@ssrobins: Checking remote: conancenter
cmake_utils/10.0.1@ssrobins: Checking remote: conancenter
Graph root
    box2d/2.3.1@ssrobins (test package): D:\a\conan-recipes\conan-recipes\recipes\box2d\test_package\conanfile.py
Requirements
    box2d/2.3.1@ssrobins#461952414da5d1f5af578bd53f97e2e0cc5babcf - Not in remote
    cmake_utils/10.0.1@ssrobins#461952414da5d1f5af578bd53f97e2e0cc5babcf - Not in remote
-------- Computing necessary packages ----------
cmake_utils/10.0.1@ssrobins: WARN: Can't update, no package in remote
box2d/2.3.1@ssrobins: Forced build from source
Requirements
    box2d/2.3.1@ssrobins#461952414da5d1f5af578bd53f97e2e0cc5babcf:d3783581517e3aa396229673622090255a79473d - Build
    cmake_utils/10.0.1@ssrobins#461952414da5d1f5af578bd53f97e2e0cc5babcf:da39a3ee5e6b4b0d3255bfef95601890afd80709#554d1dc680233d1c133e630854d882a9 - Cache
-------- Installing packages ----------
Installing (downloading, building) binaries...
cmake_utils/10.0.1@ssrobins: Already installed!
box2d/2.3.1@ssrobins: Calling source() in C:\Users\runneradmin\.conan2\p\ac46809199461bab\s
Downloading v2.3.1.tar.gz
box2d/2.3.1@ssrobins: Copying sources to build folder
box2d/2.3.1@ssrobins: Building your package in C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b
box2d/2.3.1@ssrobins: Generator 'CMakeDeps' calling 'generate()'
box2d/2.3.1@ssrobins: Calling generate()
box2d/2.3.1@ssrobins: Aggregating env generators
box2d/2.3.1@ssrobins: Calling build()
box2d/2.3.1@ssrobins: CMake command: cmake -G "Ninja Multi-Config" -DCMAKE_TOOLCHAIN_FILE="C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/b/build/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b"
box2d/2.3.1@ssrobins: RUN: cmake -G "Ninja Multi-Config" -DCMAKE_TOOLCHAIN_FILE="C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/b/build/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b"
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.2.4
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64_x86'
-- Using Conan toolchain: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/b/build/conan_toolchain.cmake
-- Conan toolchain: C++ Standard 17 with extensions OFF
-- The C compiler identification is MSVC 19.32.31329.0
-- The CXX compiler identification is MSVC 19.32.31329.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.32.31326/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.32.31326/bin/Hostx64/x86/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Target declared 'cmake_utils::cmake_utils'
CMake Deprecation Warning at source/Box2D/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.
  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/b/build
box2d/2.3.1@ssrobins: CMake command: cmake --build "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\build" --config Debug -- -j2
box2d/2.3.1@ssrobins: RUN: cmake --build "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\build" --config Debug -- -j2
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.2.4
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64_x86'
[1/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2MouseJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2MouseJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[2/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2MotorJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2MotorJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[3/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2CircleContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2CircleContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[4/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2PrismaticJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2PrismaticJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[5/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2RevoluteJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2RevoluteJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[6/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2PulleyJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2PulleyJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[7/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2RopeJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2RopeJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[8/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2WeldJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2WeldJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[9/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2EdgeAndPolygonContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2EdgeAndPolygonContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[10/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2WheelJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2WheelJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[11/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2StackAllocator.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2StackAllocator.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[12/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ChainAndCircleContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2ChainAndCircleContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[13/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ContactSolver.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2ContactSolver.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[14/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2EdgeAndCircleContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2EdgeAndCircleContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[15/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2Contact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2Contact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[16/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2PolygonAndCircleContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2PolygonAndCircleContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[17/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2ContactManager.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2ContactManager.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[18/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2PolygonContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2PolygonContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[19/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Body.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2Body.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[20/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Fixture.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2Fixture.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[21/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ChainAndPolygonContact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Contacts\b2ChainAndPolygonContact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[22/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Island.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2Island.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[23/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2BlockAllocator.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2BlockAllocator.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[24/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Draw.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2Draw.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[25/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2WorldCallbacks.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2WorldCallbacks.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[26/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Math.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2Math.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[27/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2World.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\b2World.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[28/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Settings.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2Settings.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[29/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2CircleShape.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\Shapes\b2CircleShape.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[30/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2PolygonShape.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\Shapes\b2PolygonShape.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[31/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2EdgeShape.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\Shapes\b2EdgeShape.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[32/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2ChainShape.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\Shapes\b2ChainShape.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[33/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollideCircle.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2CollideCircle.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[34/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2BroadPhase.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2BroadPhase.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[35/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2Collision.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2Collision.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[36/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollidePolygon.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2CollidePolygon.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[37/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollideEdge.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2CollideEdge.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[38/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2DynamicTree.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2DynamicTree.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[39/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2Distance.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2Distance.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[40/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2DistanceJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2DistanceJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[41/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2FrictionJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2FrictionJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[42/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2GearJoint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2GearJoint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[43/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2Joint.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Dynamics\Joints\b2Joint.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[44/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2TimeOfImpact.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Collision\b2TimeOfImpact.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[45/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Rope\b2Rope.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Rope\b2Rope.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[46/47] C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\cl.exe   /TP -DCMAKE_INTDIR=\"Debug\" -IC:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\.. /DWIN32 /D_WINDOWS /GR /EHsc /Zi /Ob0 /Od /RTC1 -MTd /MP /MTd /permissive- /ZI -std:c++17 /showIncludes /Fosource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Timer.cpp.obj /Fdsource\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Box2D.pdb /FS -c C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\source\Box2D\Box2D\Common\b2Timer.cpp
Microsoft (R) C/C++ Optimizing Compiler Version 19.32.31329 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
cl : Command line warning D9025 : overriding '/Zi' with '/ZI'
[47/47] cmd.exe /C "cd . && C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1432~1.313\bin\Hostx64\x86\lib.exe  /machine:X86 /out:source\Box2D\Box2D\Debug\Box2Dd.lib source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2DistanceJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2FrictionJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2GearJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2Joint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2MotorJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2MouseJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2PrismaticJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2PulleyJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2RevoluteJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2RopeJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2WeldJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Joints\b2WheelJoint.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2CircleContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2Contact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ContactSolver.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2PolygonAndCircleContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2EdgeAndCircleContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2EdgeAndPolygonContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ChainAndCircleContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2ChainAndPolygonContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\Contacts\b2PolygonContact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Body.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2ContactManager.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Fixture.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2Island.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2World.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Dynamics\b2WorldCallbacks.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2BlockAllocator.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Draw.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Math.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Settings.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2StackAllocator.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Common\b2Timer.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2CircleShape.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2EdgeShape.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2ChainShape.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\Shapes\b2PolygonShape.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2BroadPhase.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollideCircle.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollideEdge.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2CollidePolygon.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2Collision.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2Distance.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2DynamicTree.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Collision\b2TimeOfImpact.cpp.obj source\Box2D\Box2D\CMakeFiles\Box2D.dir\Debug\Rope\b2Rope.cpp.obj  && cd ."
Microsoft (R) Library Manager Version 14.32.31329.0
Copyright (C) Microsoft Corporation.  All rights reserved.
box2d/2.3.1@ssrobins: RUN: ctest -C Debug --output-on-failure
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.2.4
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64_x86'
Test project C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/b/build
No tests were found!!!
box2d/2.3.1@ssrobins: CMake command: cmake --install "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\build" --config Debug --prefix "C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p"
box2d/2.3.1@ssrobins: RUN: cmake --install "C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\build" --config Debug --prefix "C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p"
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.2.4
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64_x86'
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Box2D.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/b2BroadPhase.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/b2Collision.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/b2Distance.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/b2DynamicTree.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/b2TimeOfImpact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/Shapes/b2CircleShape.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/Shapes/b2EdgeShape.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/Shapes/b2ChainShape.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/Shapes/b2PolygonShape.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Collision/Shapes/b2Shape.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2BlockAllocator.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2Draw.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2GrowableStack.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2Math.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2Settings.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2StackAllocator.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Common/b2Timer.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2Body.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2ContactManager.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2Fixture.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2Island.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2TimeStep.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2World.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/b2WorldCallbacks.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2CircleContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2Contact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2ContactSolver.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2PolygonAndCircleContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2EdgeAndCircleContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2EdgeAndPolygonContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2ChainAndCircleContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2ChainAndPolygonContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Contacts/b2PolygonContact.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2DistanceJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2FrictionJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2GearJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2Joint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2MotorJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2MouseJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2PrismaticJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2PulleyJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2RevoluteJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2RopeJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2WeldJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Dynamics/Joints/b2WheelJoint.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/include/Box2D/Rope/b2Rope.h
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/lib/Box2Dd.lib
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/lib/Box2D/Box2D-targets.cmake
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/lib/Box2D/Box2D-targets-debug.cmake
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/lib/cmake/Box2D/Box2DConfig.cmake
-- Installing: C:/Users/runneradmin/.conan2/p/tmp/eb5cc8e93d895b84/p/lib/cmake/Box2D/UseBox2D.cmake
box2d/2.3.1@ssrobins: Package 'd3783581517e3aa396229673622090255a79473d' built
box2d/2.3.1@ssrobins: Build folder C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\b\build
box2d/2.3.1@ssrobins: Generated conaninfo.txt
box2d/2.3.1@ssrobins: Generating the package
box2d/2.3.1@ssrobins: Temporary package folder C:\Users\runneradmin\.conan2\p\tmp\eb5cc8e93d895b84\p
box2d/2.3.1@ssrobins: Calling package()
box2d/2.3.1@ssrobins: Copied 1 '.pdb' file: Box2D.pdb
box2d/2.3.1@ssrobins package(): Packaged 47 '.h' files
box2d/2.3.1@ssrobins package(): Packaged 1 '.pdb' file: Box2D.pdb
box2d/2.3.1@ssrobins package(): Packaged 1 '.lib' file: Box2Dd.lib
box2d/2.3.1@ssrobins package(): Packaged 4 '.cmake' files: Box2D-targets-debug.cmake, Box2D-targets.cmake, Box2DConfig.cmake, UseBox2D.cmake
box2d/2.3.1@ssrobins: Package 'd3783581517e3aa396229673622090255a79473d' created
box2d/2.3.1@ssrobins: Created package revision 06dc5288485f9814bcb2b1e1de4b1208
box2d/2.3.1@ssrobins: Full package reference: box2d/2.3.1@ssrobins#461952414da5d1f5af578bd53f97e2e0cc5babcf:d3783581517e3aa396229673622090255a79473d#06dc5288485f9814bcb2b1e1de4b1208
ERROR: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\runneradmin\\.conan2\\p\\tmp\\eb5cc8e93d895b84\\b\\build'
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 825, in move
    os.rename(src, real_dst)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\runneradmin\\.conan2\\p\\tmp\\eb5cc8e93d895b84' -> 'C:\\Users\\runneradmin\\.conan2\\p\\633f3c5ce311a547'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\cli\cli.py", line 163, in run
    command.run(self._conan_api, self._commands[command_argument].parser, args[0][1:])
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\cli\command.py", line 157, in run
    info = self._method(conan_api, parser, *args)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\cli\commands\create.py", line 103, in create
    conan_api.install.install_binaries(deps_graph=deps_graph, remotes=remotes, update=args.update)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conan\api\subapi\__init__.py", line 21, in wrapper
    return f(subapi, *args, **kwargs)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conan\api\subapi\install.py", line 29, in install_binaries
    installer.install(deps_graph)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\client\installer.py", line 225, in install
    self._handle_package(package, install_reference)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\client\installer.py", line 275, in _handle_package
    self._handle_node_build(package, package_layout)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\client\installer.py", line 353, in _handle_node_build
    self._cache.assign_prev(pkg_layout)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conans\client\cache\cache.py", line 66, in assign_prev
    return self._data_cache.assign_prev(layout)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\site-packages\conan\cache\cache.py", line [187](https://github.com/ssrobins/conan-recipes/runs/7057105287?check_suite_focus=true#step:10:188), in assign_prev
    shutil.move(self._full_path(layout.base_folder), full_path)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 843, in move
    rmtree(src)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 759, in rmtree
    return _rmtree_unsafe(path, onerror)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 624, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 624, in _rmtree_unsafe
    _rmtree_unsafe(fullname, onerror)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 633, in _rmtree_unsafe
    onerror(os.rmdir, path, sys.exc_info())
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\shutil.py", line 631, in _rmtree_unsafe
    os.rmdir(path)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\runneradmin\\.conan2\\p\\tmp\\eb5cc8e93d895b84\\b\\build'
Traceback (most recent call last):
  File "D:\a\conan-recipes\conan-recipes\scripts\conan_create_all.py", line 10, in <module>
    conan_create(f"{recipe_path}/box2d")
  File "D:\a\conan-recipes\conan-recipes\scripts\conan_create.py", line 38, in conan_create
    subprocess.run(conan_create, cwd=recipe_path, shell=True, check=True)
  File "C:\hostedtoolcache\windows\Python\3.10.5\x64\lib\subprocess.py", line [524](https://github.com/ssrobins/conan-recipes/runs/7057105287?check_suite_focus=true#step:10:525), in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'conan create --update --user ssrobins . -pr:h=D:\a\conan-recipes\conan-recipes\scripts/../profiles/windows_x86 -pr:b=D:\a\conan-recipes\conan-recipes\scripts/../profiles/windows_x64 -pr:b=D:\a\conan-recipes\conan-recipes\scripts/../profiles/default -s build_type=Debug' returned non-zero exit status 1.
Error: Process completed with exit code 1.
```

--- a/conan/api/subapi/install.py
+++ b/conan/api/subapi/install.py
@@ -113,8 +113,7 @@ def full_deploy(conanfile, output_folder):
         if arch:
             folder_name = os.path.join(folder_name, arch)
         new_folder = os.path.join(output_folder, folder_name)
-        if os.path.isdir(new_folder):
-            rmdir(new_folder)
+        rmdir(new_folder)
         shutil.copytree(dep.package_folder, new_folder)
         dep.set_deploy_folder(new_folder)
 
@@ -134,7 +133,6 @@ def direct_deploy(conanfile, output_folder):
     # dependencies
     for dep in conanfile.dependencies.filter({"direct": True}).values():
         new_folder = os.path.join(output_folder, dep.ref.name)
-        if os.path.isdir(new_folder):
-            rmdir(new_folder)
+        rmdir(new_folder)
         shutil.copytree(dep.package_folder, new_folder)
         dep.set_deploy_folder(new_folder)

--- a/conan/cache/cache.py
+++ b/conan/cache/cache.py
@@ -1,6 +1,5 @@
 import hashlib
 import os
-import shutil
 
 from conan.cache.conan_reference_layout import RecipeLayout, PackageLayout
 # TODO: Random folders are no longer accessible, how to get rid of them asap?
@@ -12,7 +11,7 @@ from conans.errors import ConanReferenceAlreadyExistsInDB, ConanReferenceDoesNot
 from conans.model.package_ref import PkgReference
 from conans.model.recipe_ref import RecipeReference
 from conans.util.dates import revision_timestamp_now
-from conans.util.files import rmdir
+from conans.util.files import rmdir, renamedir
 
 
 class DataCache:
@@ -176,12 +175,9 @@ class DataCache:
         new_path = self._get_path(pref)
 
         full_path = self._full_path(new_path)
-        if os.path.exists(full_path):
-            try:
-                rmdir(full_path)
-            except Exception:
-                raise ConanException(f"Couldn't remove folder, might be busy or open: {full_path}")
-        shutil.move(self._full_path(layout.base_folder), full_path)
+        rmdir(full_path)
+
+        renamedir(self._full_path(layout.base_folder), full_path)
         layout._base_folder = os.path.join(self.base_folder, new_path)
 
         build_id = layout.build_id
@@ -214,11 +210,8 @@ class DataCache:
         # TODO: cache2.0 probably we should not check this and move to other place or just
         #  avoid getting here if old and new paths are the same
         full_path = self._full_path(new_path)
-        try:
-            rmdir(full_path)
-        except Exception:
-            raise ConanException(f"Couldn't remove folder, might be busy or open: {full_path}")
-        shutil.move(self._full_path(layout.base_folder), full_path)
+        rmdir(full_path)
+        renamedir(self._full_path(layout.base_folder), full_path)
         layout._base_folder = os.path.join(self.base_folder, new_path)
 
         # Wait until it finish to really update the DB

--- a/conan/cache/conan_reference_layout.py
+++ b/conan/cache/conan_reference_layout.py
@@ -27,10 +27,7 @@ class LayoutBase:
         return self._base_folder
 
     def remove(self):
-        try:
-            rmdir(self.base_folder)
-        except OSError as e:
-            raise ConanException(f"Couldn't remove folder {self.base_folder}: {str(e)}")
+        rmdir(self.base_folder)
 
 
 class RecipeLayout(LayoutBase):
@@ -76,12 +73,7 @@ class RecipeLayout(LayoutBase):
 
     def sources_remove(self):
         src_folder = self.source()
-        try:
-            rmdir(src_folder)  # This will remove the shortened path too if exists
-        except OSError as e:
-            raise ConanException("%s\n\nFolder: %s\n"
-                                 "Couldn't remove folder, might be busy or open\n"
-                                 "Close any app using it, and retry" % (src_folder, str(e)))
+        rmdir(src_folder)
 
     def export_remove(self):
         export_folder = self.export()
@@ -138,21 +130,13 @@ class PackageLayout(LayoutBase):
         return is_dirty(self.package())
 
     def build_remove(self):
-        try:
-            rmdir(self.build())
-        except OSError as e:
-            raise ConanException(f"Couldn't remove folder {self.build()}: {str(e)}")
+        rmdir(self.build())
 
     # TODO: cache2.0 locks
     def package_remove(self):
         # Here we could validate and check we own a write lock over this package
         tgz_folder = self.download_package()
         rmdir(tgz_folder)
-        try:
-            rmdir(self.package())
-        except OSError as e:
-            raise ConanException("%s\n\nFolder: %s\n"
-                                 "Couldn't remove folder, might be busy or open\n"
-                                 "Close any app using it, and retry" % (self.package(), str(e)))
+        rmdir(self.package())
         if is_dirty(self.package()):
             clean_dirty(self.package())

--- a/conans/client/installer.py
+++ b/conans/client/installer.py
@@ -94,7 +94,7 @@ class _PackageBuilder(object):
     @staticmethod
     def _copy_sources(conanfile, source_folder, build_folder):
         # Copies the sources to the build-folder, unless no_copy_source is defined
-        _remove_folder_raising(build_folder)
+        rmdir(build_folder)
         if not getattr(conanfile, 'no_copy_source', False):
             conanfile.output.info('Copying sources to build folder')
             try:
@@ -181,14 +181,6 @@ class _PackageBuilder(object):
                 raise exc
 
         return node.pref
-
-
-def _remove_folder_raising(folder):
-    try:
-        rmdir(folder)
-    except OSError as e:
-        raise ConanException("%s\n\nCouldn't remove folder, might be busy or open\n"
-                             "Close any app using it, and retry" % str(e))
 
 
 def call_system_requirements(conanfile):

--- a/conans/client/source.py
+++ b/conans/client/source.py
@@ -56,14 +56,7 @@ def config_source(export_source_folder, conanfile, hook_manager):
 
     def remove_source():
         conanfile.output.warning("This can take a while for big packages")
-        try:
-            rmdir(conanfile.folders.base_source)
-        except BaseException as e_rm:
-            msg = str(e_rm)
-            conanfile.output.error("Unable to remove source folder %s\n%s"
-                                   % (conanfile.folders.base_source, msg))
-            conanfile.output.warning("**** Please delete it manually ****")
-            raise ConanException("Unable to remove source folder")
+        rmdir(conanfile.folders.base_source)
 
     if is_dirty(conanfile.folders.base_source):
         conanfile.output.warning("Trying to remove corrupted source folder")

--- a/conans/util/files.py
+++ b/conans/util/files.py
@@ -2,6 +2,7 @@ import errno
 import gzip
 import hashlib
 import os
+import platform
 import shutil
 import stat
 import tarfile
@@ -241,37 +242,58 @@ def _change_permissions(func, path, exc_info):
         raise OSError("Cannot change permissions for {}! Exception info: {}".format(path, exc_info))
 
 
-def rmdir(path):
-    if not os.path.isdir(path):
-        return
+if platform.system() == "Windows":
+    def rmdir(path):
+        if not os.path.isdir(path):
+            return
 
-    retries = 3
-    delay = 0.5
-    for i in range(retries):
+        retries = 3
+        delay = 0.5
+        for i in range(retries):
+            try:
+                shutil.rmtree(path, onerror=_change_permissions)
+                break
+            except OSError as err:
+                if i == retries - 1:
+                    raise ConanException(f"Couldn't remove folder: {path}\n{str(err)}\n"
+                                         "Folder might be busy or open. "
+                                         "Close any app using it and retry.")
+                time.sleep(delay)
+
+
+    def renamedir(old_path, new_path):
+        retries = 3
+        delay = 0.5
+        for i in range(retries):
+            try:
+                shutil.move(old_path, new_path)
+                break
+            except OSError as err:
+                if i == retries - 1:
+                    raise ConanException(f"Couldn't move folder: {old_path}->{new_path}\n"
+                                         f"{str(err)}\n"
+                                         "Folder might be busy or open. "
+                                         "Close any app using it and retry.")
+                time.sleep(delay)
+else:
+    def rmdir(path):
+        if not os.path.isdir(path):
+            return
         try:
             shutil.rmtree(path, onerror=_change_permissions)
-            break
         except OSError as err:
-            if i == retries - 1:
-                raise ConanException(f"Couldn't remove folder: {path}\n{str(err)}\n"
-                                     "Folder might be busy or open. "
-                                     "Close any app using it and retry.")
-            time.sleep(delay)
+            raise ConanException(f"Couldn't remove folder: {path}\n{str(err)}\n"
+                                 "Folder might be busy or open. "
+                                 "Close any app using it and retry.")
 
-
-def renamedir(old_path, new_path):
-    retries = 3
-    delay = 0.5
-    for i in range(retries):
+    def renamedir(old_path, new_path):
         try:
             shutil.move(old_path, new_path)
-            break
         except OSError as err:
-            if i == retries - 1:
-                raise ConanException(f"Couldn't move folder: {old_path}->{new_path}\n{str(err)}\n"
-                                     "Folder might be busy or open. "
-                                     "Close any app using it and retry.")
-            time.sleep(delay)
+            raise ConanException(
+                f"Couldn't move folder: {old_path}->{new_path}\n{str(err)}\n"
+                "Folder might be busy or open. "
+                "Close any app using it and retry.")
 
 
 def remove(path):

--- a/conans/util/files.py
+++ b/conans/util/files.py
@@ -6,11 +6,12 @@ import shutil
 import stat
 import tarfile
 import tempfile
-
+import time
 
 from os.path import abspath, join as joinpath, realpath
 from contextlib import contextmanager
 
+from conans.errors import ConanException
 
 _DIRTY_FOLDER = ".dirty"
 
@@ -241,12 +242,36 @@ def _change_permissions(func, path, exc_info):
 
 
 def rmdir(path):
-    try:
-        shutil.rmtree(path, onerror=_change_permissions)
-    except OSError as err:
-        if err.errno == errno.ENOENT:
-            return
-        raise
+    if not os.path.isdir(path):
+        return
+
+    retries = 3
+    delay = 0.5
+    for i in range(retries):
+        try:
+            shutil.rmtree(path, onerror=_change_permissions)
+            break
+        except OSError as err:
+            if i == retries - 1:
+                raise ConanException(f"Couldn't remove folder: {path}\n{str(err)}\n"
+                                     "Folder might be busy or open. "
+                                     "Close any app using it and retry.")
+            time.sleep(delay)
+
+
+def renamedir(old_path, new_path):
+    retries = 3
+    delay = 0.5
+    for i in range(retries):
+        try:
+            shutil.move(old_path, new_path)
+            break
+        except OSError as err:
+            if i == retries - 1:
+                raise ConanException(f"Couldn't move folder: {old_path}->{new_path}\n{str(err)}\n"
+                                     "Folder might be busy or open. "
+                                     "Close any app using it and retry.")
+            time.sleep(delay)
 
 
 def remove(path):

